extensions/functions_arithmetic.yaml

```diff
@@ -1563,6 +1563,43 @@ aggregate_functions:
             values: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
         nullability: DECLARED_OUTPUT
         return: fp64?
+  - name: "first"
+    description: >-
+      First value in a set of rows.
+
+      The `null_handling` option determines whether or not null values will be recognized by the function.
+      If `null_handling` is set to `IGNORE_NULLS`, the function will look for the first non-null value.
+      If set to `ACCEPT_NULLS`, the function will return the first value it sees, even if null.
+
+      If all values are null or there are no values, returns null.
+    impls:
+      - args:
+          - name: x
+            value: any
```

Comment:
minor: if we replace `any` with `any1` in the definition, it enforces that they are all the same type.

See https://substrait.io/extensions/#any-types

Reply:
ah right, I knew that but forgot, thanks! 3dc5d18bf37ce08a4aa2faec2f07877261290a4b